tests/ord-aggregator/tests/handler_test.go

```diff
@@ -398,3 +411,20 @@ func storeMappingBetweenORDAndInternalBundleID(t *testing.T, respBody string, nu
 
 	return ordAndInternalIDsMapping
 }
+
+func getGlobalResourcesNumber(ctx context.Context, httpClient *http.Client) (int, int) {
+	certCache, err := certloader.StartCertLoader(ctx, testConfig.CertLoaderConfig)
+	if err != nil {
+		log.C(ctx).Errorf("Failed to initialize certificate loader %v", err)
+		return 0, 0
+	}
```

Comment:
Fail the test here as well, better early with error with context instead of later.